config/vufind/config.ini

```diff
@@ -866,6 +866,10 @@ coverimagesCache = true
 ; default; to enable it, just uncomment the line below.
 ;ajaxcovers = true
 
+; This setting controls whether a fallback template is rendered in case no cover
+; image could be loaded
```

Comment:
As implemented, it now looks like this will only work in conjunction with AJAX covers. Is that the case? If so, it should probably be documented here. (I imagine that a non-AJAX implementation would be possible, but it could have serious performance implications, so leaving it to AJAX is probably for the best).